hack/e2e/run-rp-and-e2e.sh

```diff
@@ -37,6 +37,42 @@ kill_rp(){
     wait $rppid
 }
 
+validate_portal_running() {
+    echo "########## ？Checking Admin Portal Status ##########"
+    ELAPSED=0
+    while true; do
+        http_code=$(curl -k -s -o /dev/null -w '%{http_code}' https://localhost:8444/api/info || true)
```

Comment:
The ` || true` is very javascripty in this context :) and not required.

Reply:
Fixed!